apps/spark/src/main/java/com/linkedin/openhouse/jobs/client/TablesClient.java

```diff
@@ -64,7 +71,8 @@ private Optional<RetentionConfig> getTableRetention(GetTableResponseBody respons
     if (response.getTimePartitioning() != null) {
       columnName = response.getTimePartitioning().getColumnName();
     } else {
-      columnName = policies.getRetention().getColumnPattern().getColumnName();
+      columnName =
```

Comment:
Is this a bug fix not relevant to this PR?

Reply:
Ideally a separate PR to show before and after symptoms.

Reply:
But given this is small, feel free to update the testing done section for this PR.

Reply:
Removing those unnecessary changes from this PR